src/network/core/network_game_info.h

```diff
@@ -92,8 +95,9 @@ enum NewGRFSerializationType {
  */
 struct NetworkServerGameInfo {
 	GRFConfig *grfconfig;        ///< List of NewGRF files used
-	TimerGameCalendar::Date start_date; ///< When the game started
-	TimerGameCalendar::Date game_date;  ///< Current date
+	TimerGameCalendar::Date calendar_start; ///< When the game started.
+	TimerGameCalendar::Date calendar_date;  ///< Current calendar date.
+	TimerGameTick::TickCounter ticks_running;  ///< Amount of ticks the game has been running.
```

Comment:
Not sure whether the comment of `calendar_date` should align with `calendar_start`. Though the two spaces for `ticks_running` seems too much.
```suggestion
	TimerGameCalendar::Date calendar_date; ///< Current calendar date.
	TimerGameTick::TickCounter ticks_running; ///< Amount of ticks the game has been running.
```